### ISSUE TITLE
NoClassDefFoundError: Could not initialize class com.unboundid.util.Debug

### ISSUE THREAD

**Issue by hendry-lim** [elastic/elasticsearch#68838]
**Elasticsearch version** (`bin/elasticsearch --version`): 7.11.0 - Docker

**Description of the problem including expected versus actual behavior**:
```
{"type": "server", "timestamp": "2021-02-10T15:32:51,596Z", "level": "ERROR", "component": "o.e.b.ElasticsearchUncaughtExceptionHandler", "cluster.name": "docker-cluster", "node.name": "es02", "message": "fatal error in thread [main], exiting", 
"stacktrace": ["java.lang.NoClassDefFoundError: Could not initialize class com.unboundid.util.Debug",
"at com.unboundid.ldap.sdk.LDAPConnectionOptions.getSystemProperty(LDAPConnectionOptions.java:2278) ~[?:?]",
"at com.unboundid.ldap.sdk.LDAPConnectionOptions.<clinit>(LDAPConnectionOptions.java:213) ~[?:?]",
"at org.elasticsearch.xpack.security.authc.ldap.support.SessionFactory.connectionOptions(SessionFactory.java:122) ~[?:?]",
"at org.elasticsearch.xpack.security.authc.ldap.support.SessionFactory.serverSet(SessionFactory.java:203) ~[?:?]",
"at org.elasticsearch.xpack.security.authc.ldap.support.SessionFactory.<init>(SessionFactory.java:81) ~[?:?]",
"at org.elasticsearch.xpack.security.authc.ldap.PoolingSessionFactory.<init>(PoolingSessionFactory.java:62) ~[?:?]",
"at org.elasticsearch.xpack.security.authc.ldap.ActiveDirectorySessionFactory.<init>(ActiveDirectorySessionFactory.java:68) ~[?:?]",
"at org.elasticsearch.xpack.security.authc.ldap.LdapRealm.sessionFactory(LdapRealm.java:89) ~[?:?]",
"at org.elasticsearch.xpack.security.authc.ldap.LdapRealm.<init>(LdapRealm.java:68) ~[?:?]",
"at org.elasticsearch.xpack.security.authc.InternalRealms.lambda$getFactories$2(InternalRealms.java:113) ~[?:?]",
"at org.elasticsearch.xpack.security.authc.Realms.initRealms(Realms.java:214) ~[?:?]",
"at org.elasticsearch.xpack.security.authc.Realms.<init>(Realms.java:76) ~[?:?]",
"at org.elasticsearch.xpack.security.Security.createComponents(Security.java:480) ~[?:?]",
"at org.elasticsearch.xpack.security.Security.createComponents(Security.java:412) ~[?:?]",
"at org.elasticsearch.node.Node.lambda$new$16(Node.java:560) ~[elasticsearch-7.11.0.jar:7.11.0]",
"at java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:271) ~[?:?]",
"at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625) ~[?:?]",
"at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) ~[?:?]",
"at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474) ~[?:?]",
"at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913) ~[?:?]",
"at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?]",
"at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578) ~[?:?]",
"at org.elasticsearch.node.Node.<init>(Node.java:564) ~[elasticsearch-7.11.0.jar:7.11.0]",
"at org.elasticsearch.node.Node.<init>(Node.java:278) ~[elasticsearch-7.11.0.jar:7.11.0]",
"at org.elasticsearch.bootstrap.Bootstrap$5.<init>(Bootstrap.java:216) ~[elasticsearch-7.11.0.jar:7.11.0]",
"at org.elasticsearch.bootstrap.Bootstrap.setup(Bootstrap.java:216) ~[elasticsearch-7.11.0.jar:7.11.0]",
"at org.elasticsearch.bootstrap.Bootstrap.init(Bootstrap.java:387) ~[elasticsearch-7.11.0.jar:7.11.0]",
"at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:159) ~[elasticsearch-7.11.0.jar:7.11.0]",
"at org.elasticsearch.bootstrap.Elasticsearch.execute(Elasticsearch.java:150) ~[elasticsearch-7.11.0.jar:7.11.0]",
"at org.elasticsearch.cli.EnvironmentAwareCommand.execute(EnvironmentAwareCommand.java:75) ~[elasticsearch-7.11.0.jar:7.11.0]",
"at org.elasticsearch.cli.Command.mainWithoutErrorHandling(Command.java:116) ~[elasticsearch-cli-7.11.0.jar:7.11.0]",
"at org.elasticsearch.cli.Command.main(Command.java:79) ~[elasticsearch-cli-7.11.0.jar:7.11.0]",
"at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:115) ~[elasticsearch-7.11.0.jar:7.11.0]",
"at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:81) ~[elasticsearch-7.11.0.jar:7.11.0]"] }
fatal error in thread [main], exiting
java.lang.NoClassDefFoundError: Could not initialize class com.unboundid.util.Debug
        at com.unboundid.ldap.sdk.LDAPConnectionOptions.getSystemProperty(LDAPConnectionOptions.java:2278)
        at com.unboundid.ldap.sdk.LDAPConnectionOptions.<clinit>(LDAPConnectionOptions.java:213)
        at org.elasticsearch.xpack.security.authc.ldap.support.SessionFactory.connectionOptions(SessionFactory.java:122)
        at org.elasticsearch.xpack.security.authc.ldap.support.SessionFactory.serverSet(SessionFactory.java:203)
        at org.elasticsearch.xpack.security.authc.ldap.support.SessionFactory.<init>(SessionFactory.java:81)
        at org.elasticsearch.xpack.security.authc.ldap.PoolingSessionFactory.<init>(PoolingSessionFactory.java:62)
        at org.elasticsearch.xpack.security.authc.ldap.ActiveDirectorySessionFactory.<init>(ActiveDirectorySessionFactory.java:68)
        at org.elasticsearch.xpack.security.authc.ldap.LdapRealm.sessionFactory(LdapRealm.java:89)
        at org.elasticsearch.xpack.security.authc.ldap.LdapRealm.<init>(LdapRealm.java:68)
        at org.elasticsearch.xpack.security.authc.InternalRealms.lambda$getFactories$2(InternalRealms.java:113)
        at org.elasticsearch.xpack.security.authc.Realms.initRealms(Realms.java:214)
        at org.elasticsearch.xpack.security.authc.Realms.<init>(Realms.java:76)
        at org.elasticsearch.xpack.security.Security.createComponents(Security.java:480)
        at org.elasticsearch.xpack.security.Security.createComponents(Security.java:412)
        at org.elasticsearch.node.Node.lambda$new$16(Node.java:560)
        at java.base/java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:271)
        at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625)
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
        at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578)
        at org.elasticsearch.node.Node.<init>(Node.java:564)
        at org.elasticsearch.node.Node.<init>(Node.java:278)
        at org.elasticsearch.bootstrap.Bootstrap$5.<init>(Bootstrap.java:216)
        at org.elasticsearch.bootstrap.Bootstrap.setup(Bootstrap.java:216)
        at org.elasticsearch.bootstrap.Bootstrap.init(Bootstrap.java:387)
        at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:159)
        at org.elasticsearch.bootstrap.Elasticsearch.execute(Elasticsearch.java:150)
        at org.elasticsearch.cli.EnvironmentAwareCommand.execute(EnvironmentAwareCommand.java:75)
        at org.elasticsearch.cli.Command.mainWithoutErrorHandling(Command.java:116)
        at org.elasticsearch.cli.Command.main(Command.java:79)
        at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:115)
        at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:81)
```

**Steps to reproduce**:
 1. Pull the new `7.11.0` Docker image
 2. Restart Elasticsearch container

AD settings (working fine on <= `7.10.2`):
```
xpack.security.authc.realms.active_directory.ad.order: 1
xpack.security.authc.realms.active_directory.ad.domain_name: domain
xpack.security.authc.realms.active_directory.ad.url: ldap://ip:389
```

**Comment by hendry-lim** [comment]
Will be fixed by #68872

**Comment by jasontedor** [comment]
Thanks for reporting @hendry-lim. We are working on a fix in #68872.
